metadata.txt

```diff
@@ -19,14 +19,25 @@ email=tim@kartoza.com
 # Optional items:
 
 # Uncomment the following line and add your changelog:
-# changelog=
+changelog=
+	v2.0.0:
+	- Code converted to Python 3
+	- Plugin can now be installed in QGIS 3
+	- Fixed a language translation issue
+	- Processing can correctly be applied to multilinestring files
+	- Error submitting now correctly disables
+	- Sphinx documenation updated
+	- Updates to the test-suite
+	- GitHub actions
```

Comment:
I'm not sure we should mention about the `Github actions` here as the changelog should be for plugin features specific changes.